cypress/e2e/shared/dashboardsView.test.ts

```diff
@@ -287,224 +289,371 @@ describe('Dashboard', () => {
+                  .eq(2)
+                  .trigger('mouseover')
+                // toggle the variable dropdown in the VEO cell dashboard
+                cy.getByTestID('toolbar-popover--contents').within(() => {
+                  cy.getByTestID('variable-dropdown--button').click()
+                  // select 1st value in cell
+                  cy.getByTestID('variable-dropdown--item')
+                    .first()
+                    .click()
+                })
+                // save cell
+                cy.getByTestID('save-cell--button').click()
+
+                // selected value in cell context is 1st value
+                cy.window()
+                  .pipe(getSelectedVariable(dashboard.id, 2))
+                  .should('equal', 'v1')
+
+                // selected value in dashboard is 1st value
+                cy.getByTestID(
+                  `variable-dropdown-input-typeAhead--${mapTypeVarName}`
+                ).should('have.value', 'k1')
+                cy.window()
+                  .pipe(getSelectedVariable(dashboard.id, 2))
+                  .should('equal', 'v1')
+
+                cy.getByTestID('cell-context--toggle').click()
+                cy.getByTestID('cell-context--delete').click()
+                cy.getByTestID('cell-context--delete-confirm').click()
+
+                // create a new cell
+                cy.getByTestID('add-cell--button').click()
+                cy.getByTestID('switch-to-script-editor').should('be.visible')
+                cy.getByTestID('switch-to-script-editor').click()
+
+                // query for data
+                cy.getByTestID('flux-editor')
+                  .should('be.visible')
+                  .click()
+                  .focused()
+                  .clear()
+                  .type(
+                    `from(bucket: v.bucketsCSV)
+|> range(start: v.timeRangeStart, stop: v.timeRangeStop)
+|> filter(fn: (r) => r["_measurement"] == "m")
+|> filter(fn: (r) => r["_field"] == "v")
+|> filter(fn: (r) => r["tk1"] == "tv1")
+|> aggregateWindow(every: v.windowPeriod, fn: max)
+|> yield(name: "max")`,
+                    {force: true, delay: 1}
+                  )
+
+                // `bucketOne` should not exist nor have data written to it
+                cy.getByTestID('save-cell--button').click()
+                cy.getByTestID('empty-graph-error').contains(`${bucketOne}`)
+
+                // select default bucket that has data
+                cy.getByTestID('variable-dropdown--button')
+                  .eq(0)
+                  .click()
+                cy.get(`#${defaultBucket}`).click()
+
+                // assert visualization appears
+                cy.getByTestID('giraffe-layer-line').should('exist')
               })
-              // save cell
-              cy.getByTestID('save-cell--button').click()
+            })
+          })
+        })
+      })
+    })
+
+    it('dependent variables reload properly', () => {
```

Comment:
jest (and any testing framework that is an ancestor of rspect, which jest is through mocha and jasmine) is designed so that description blocks and test/it blocks flow like english sentences. this is done by considering the `it` function to be a noun in the sentence of the string that describes the test. so this description reads like:

>it dependent variables reload properly

when it'll flow better with something like:

```ts
it('reloads dependent variables properly')
```

small nit, but helps with readability and maybe clarifying what the behavior we're testing is